### DISCUSSION TITLE
Kein Signal von VR70 an ebusd V5

### DISCUSSION THREAD

**Discussion by nobbilie** [john30/ebusd#1191] [Q&A] [answered]
Hello everyone,

I am now using the V5 adapter on a Vaillant EcoTecPlus in conjunction with a Vaillant Recovair 260/4 ventilation system and a Vaillant VIH R120 water heater.
Everything is connected to a VR 70 and controlled by a Multimatic (VRC?) 700.
The ventilation system was already connected to the red eBus in the VR70 - I temporarily unplugged it and connected the eBus V5.
The setup worked to the extent that the adapter was recognized and started by Home Assistant with MQTT.
Unfortunately, however, no signal arrives at the V5 adapter from the VR70.
Am I making a mistake and the connection must be made somewhere else?

![Unbenannt1](https://github.com/john30/ebusd/assets/154513250/20d71d4a-8a2e-4d43-a7b3-dbf5f4fd6bb3)

Kind Regards

**Comment by Koky05** [comment, marked as answer]
Connect both ventilation system and eBus V5 into same place red eBus in the VR70.

**Comment by JonesGW-MGD** [comment]
As suggested by @Koky05  you need all devices to be on the bus - they connect in parallel, with no polarity to observe.  (Of course this means that depending on the physical location of your devices, you are not restricted to simply connecting to the VR71 wiring centre - the Recovair has an accessible ebus connector for example or the VRC700 if it is a wired, free-standing unit (I am not familiar with the uniSTOR - I suspect that is just controlled by a NTC (Vaillant VR10) from the VR70).  The VR71 is probably the best place though - and it is best to keep all ebus connections as short as possible.

If there are two many wires to safely secure in the provided terminals you can of course use some Wago connectors to extend the bus connections - they will fit inside the VR71 easily

**Comment by nobbilie** [comment]
Oh man, so simple and yet so effective.
Thank you very much, that was the solution.
